tests/arithmetics/execute.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
should be called with the jest CLI instead of another shell script.

might need to add *.spec.ts at the end of the file and update the jest config for pattern recognition for the tests.

If there need to be a server running, we should figure that out in the test file possibly